localstack/services/awslambda/invocation/lambda_service.py

```diff
@@ -118,6 +124,27 @@ def create_function_version(self, function_version: FunctionVersion) -> None:
             self.lambda_starting_versions[qualified_arn] = version_manager
         self.task_executor.submit(version_manager.start)
 
+    def publish_version(self, function_version: FunctionVersion):
+        """
+        Synchronously create a function version (manager)
```

Comment:
Would be great if you could add here until "when" this basically blocks. Just so that the `synchronous` nature is easier to interpret here. (E.g. Function version is in a final state after execution or similar)